README.md

```diff
@@ -14,6 +14,19 @@ on building killer features and UI.
 - [Blog](http://emberjs.com/blog)
 - [Builds](http://emberjs.com/builds)
 
+###What to Use Ember.js for and When to Use It
+A javascript "framework", or "library", is a collection of javascript files comprising features and functionality that can be incorporated into a web-development project, thereby enhancing the project's capabilities and feature set. The best frameworks are modular, creating flexibility by allowing the developer to use as little or as much of the library as the project requires.
```

Comment:
This paragraph is a bit vague, and I feel like dropping it would make the section stronger as a whole.